lib/Model/Pass/CMakeLists.txt

```diff
@@ -14,4 +14,4 @@ revng_add_library_internal(
   PurgeUnnamedAndUnreachableTypes.cpp
   Verify.cpp)
 
-target_link_libraries(revngModelPasses revngABI revngModel revngPipes)
+target_link_libraries(revngModelPasses revngModel revngPipes)
```

Comment:
I changed the dependency order, now `ABI` depends on `ModelPasses` instead of the other way around.

https://github.com/revng/revng/pull/232/commits/32f695af7208d07f671a76bf145dc22dc82ca44d

This lets me use `purgeUnnamed...Types` in ABI analyses.

@aleclearmind, feel free to close if the commits look good.

Reply:
This entire file.